docs/source/guides/performance.ipynb

```diff
@@ -0,0 +1,208 @@
+   "id": "57aaa835",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    ".. note::\n",
+    "\n",
+    "    When using a persistent cluster, Featuretools publishes a copy of the ``EntitySet`` to the cluster the first time it calculates a feature matrix. Based on the ``EntitySet``'s metadata the cluster will reuse it for successive computations. This means if two ``EntitySets`` have the same metadata but different row values (e.g. new data is added to the ``EntitySet``), Featuretools won’t recopy the second ``EntitySet`` in later calls. A simple way to avoid this scenario is to use a unique ``EntitySet`` id."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cdecad1d",
+   "metadata": {},
+   "source": [
+    "#### Using the distributed dashboard\n",
+    "Dask.distributed has a web-based diagnostics dashboard that can be used to analyze the state of the workers and tasks. It can also be useful for tracking memory use or visualizing task run-times. An in-depth description of the web interface can be found [here](https://distributed.readthedocs.io/en/latest/web.html).\n",
+    "\n",
+    "![Distributed dashboard image](../_static/images/dashboard.png)\n",
+    "\n",
+    "The dashboard requires an additional python package, bokeh, to work. Once bokeh is installed, the web interface will be launched by default when a LocalCluster is created. The cluster created by featuretools when using `n_jobs` does not enable the web interface automatically. To do so, the port to launch the main web interface on must be specified in `dask_kwargs`:\n",
+    "\n",
+    "```python3\n",
+    "fm = ft.calculate_feature_matrix(features=features,\n",
+    "                                 entityset=entityset,\n",
+    "                                 cutoff_time=cutoff_time,\n",
+    "                                 n_jobs=2,\n",
+    "                                 dask_kwargs={'diagnostics_port': 8787}\n",
+    "                                 verbose=True)\n",
+    "```\n",
+    "\n",
+    "### Parallel Computation by Partitioning Data\n",
+    "As an alternative to Featuretools' parallelization, the data can be partitioned and the feature calculations run on multiple cores or a cluster using Dask or Apache Spark with PySpark. This approach may be necessary with a large pandas `EntitySet` because the current parallel implementation sends the entire `EntitySet` to each worker which may exhaust the worker memory. Dask and Spark allow Featuretools to scale to multiple cores on a single machine or multiple machines on a cluster."
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "94c74ef7",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    ".. note::\n",
+    "    Partitioning data is not necessary when using a Dask ``EntitySet``, as the Dask dataframes that make up the ``EntitySet`` are already partitioned. Partitioning is only needed when working with pandas entities."
```

Comment:
entities -> DataFrames

Reply:
We might also want to update this note to mention Koalas along with Dask.